src/Okta.Sdk/Group.cs

```diff
@@ -23,5 +23,9 @@ public Task DeleteAsync(CancellationToken cancellationToken = default(Cancellati
         /// <inheritdoc/>
         public Task<IGroup> UpdateAsync(CancellationToken cancellationToken = default(CancellationToken))
             => GetClient().Groups.UpdateGroupAsync(this, Id, cancellationToken);
+
+        /// <inheritdoc/>
+        public Task<IRole> AssignRoleAsync(IAssignRoleRequest assignRoleRequest, string disableNotifications, CancellationToken cancellationToken = default(CancellationToken))
+            => GetClient().Groups.AssignRoleToGroupAsync(assignRoleRequest, Id, (bool?)null, cancellationToken);
```

Comment:
`string disableNotifications` is not used in the function. Is this by intention?

Reply:
The method with the string has been marked as obsolete and will be removed in the next major version. We're keeping it this way for backward compatibility.